docs/apis/avalanchego/avalanchego-release-notes.md

```diff
@@ -6,7 +6,7 @@
 
 :::
 
-## v1.9.9 [View on GitHub](https://github.com/ava-labs/avalanchego/releases/tag/v1.9.9)
+## V1.9.9 [View on GitHub](https://github.com/ava-labs/avalanchego/releases/tag/v1.9.9)
```

Comment:
I intentionally did this as `v` rather than `V`. No one writes the version prefix with a capital `V`